src/coreComponents/events/EventManager.cpp

```diff
@@ -224,39 +226,48 @@ bool EventManager::run( DomainPartition & domain )
 
 void EventManager::outputTime() const
 {
+  // The formating here is a work in progress.
+  GEOS_LOG_RANK_0( GEOS_FMT( "\n"
```

Comment:
why this duplicates the output below?
i don't really like multi-line output in general, harder to filter out for no big reason

Reply:
I explained that in the description: The old timestep output is temporarily kept in order to keep compatibility with log reading scripts that are currently in use to read the GEOS outputs.

This old output is to considered as deprecated, it will be removed later.